--- /monitrc	2014-10-29 16:45:37.000000000 +0100
+++ /monitrc.new	2014-11-12 20:48:26.000000000 +0100
@@ -25,19 +25,19 @@
 ## omitted, Monit will use 'user' facility by default. If you want to log to 
 ## a standalone log file instead, specify the full path to the log file
 #
-# set logfile syslog facility log_daemon                       
+set logfile syslog facility log_daemon                       
 #
 #
 ## Set the location of the Monit lock file which stores the process id of the
 ## running Monit instance. By default this file is stored in $HOME/.monit.pid
 #
-# set pidfile /var/run/monit.pid
+set pidfile /var/run/monit.pid
 #
 ## Set the location of the Monit id file which stores the unique id for the
 ## Monit instance. The id is generated and stored on first Monit start. By 
 ## default the file is placed in $HOME/.monit.id.
 #
-# set idfile /var/.monit.id
+set idfile /tmp/.monit.id
 #
 ## Set the location of the Monit state file which saves monitoring states
 ## on each cycle. By default the file is placed in $HOME/.monit.state. If
@@ -45,7 +45,7 @@
 ## the monitoring state across reboots. If it is on temporary filesystem, the
 ## state will be lost on reboot which may be convenient in some situations.
 #
-# set statefile /var/.monit.state
+set statefile /tmp/.monit.state
 #
 ## Set the list of mail servers for alert delivery. Multiple servers may be 
 ## specified using a comma separator. If the first mail server fails, Monit 
@@ -125,12 +125,12 @@
 ## commands to a running Monit daemon. See the Monit Wiki if you want to 
 ## enable SSL for the web server. 
 #
-set httpd port 2812 and
-    use address localhost  # only accept connection from localhost
-    allow localhost        # allow localhost to connect to the server and
-    allow admin:monit      # require user 'admin' with password 'monit'
-    allow @monit           # allow users of group 'monit' to connect (rw)
-    allow @users readonly  # allow users of group 'users' to connect readonly
+#set httpd port 2812 and
+#    use address localhost  # only accept connection from localhost
+#    allow localhost        # allow localhost to connect to the server and
+#    allow admin:monit      # require user 'admin' with password 'monit'
+#    allow @monit           # allow users of group 'monit' to connect (rw)
+#    allow @users readonly  # allow users of group 'users' to connect readonly
 
 ###############################################################################
 ## Services
@@ -248,5 +248,5 @@
 ## It is possible to include additional configuration parts from other files or
 ## directories.
 #
-#  include /etc/monit.d/*
+include /etc/monit.d/*
 #
